image/rhel/konflux.Dockerfile

```diff
@@ -1,3 +1,31 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest AS rocksdb-builder-base
```

Comment:
Yes. Although it does not take that long with rocksdb build.
* main's `build-container` step with rocksdb takes 20m49s [link](https://console.redhat.com/preview/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/main-on-pull-request-ctwd9).
* main's `build-container` step without rocksdb takes 16m42s [link](https://console.redhat.com/preview/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/main-on-push-cccmf).

This step will be optimized away when we sunset rocksdb by 4.5.0 (at least, that's the plan) - https://issues.redhat.com/browse/ROX-21207

Reply:
Cool - shaving off 20% of build time seems pretty nice, but I was really just making sure there wasn't any easy way to avoid building every time.  And we should already be saving a bit of time after you combined the installer stages.